cmd/builder/metadata.yaml

```diff
@@ -0,0 +1,9 @@
+type: builder
+github_project: open-telemetry/opentelemetry-collector
+
+status:
+  class: cmd
+  stability:
+    alpha: [metrics]
+  codeowners:
+    active: []
```

Comment:
[nitpick]
```suggestion
    active: []

```

Reply:
Fixed in [8cd238d](https://github.com/open-telemetry/opentelemetry-collector/pull/12592/commits/8cd238d89f0da1f672245837ec20c032f5583792)